packages/ui-react/src/components/FeaturedShelf/FeaturedShelf.module.scss

```
@@ -0,0 +1,308 @@
+@use '@jwp/ott-ui-react/src/styles/mixins/responsive';
+@use '@jwp/ott-ui-react/src/styles/variables';
+@use '@jwp/ott-ui-react/src/styles/theme';
+@use '@jwp/ott-ui-react/src/styles/mixins/typography';
+
+.shelf {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-start;
+  width: 100%;
+  height: 70vh;
+  min-height: 350px;
+  margin: 0 0 24px 0;
+  padding: 100px 56px;
+
+  color: var(--primary-color);
+  font-family: var(--body-alt-font-family);
+  font-size: 18px;
+  line-height: 1.5em;
+  letter-spacing: 0.5px;
+  background-color: var(--featured-shelf-background-color);
+
+  &:hover .chevron:not(:disabled) {
+    opacity: 0.8;
+  }
+
+  @include responsive.mobile-and-small-tablet() {
+    height: 67vh;
+    padding: 0;
+  }
+}
+
+.loadingTitle {
+  width: 35%;
+  min-height: 28px;
+  margin-bottom: 12px;
+  background-color: theme.$card-loading-bg-color;
+  border-radius: 5px;
+}
+
+.title {
+  display: -webkit-box;
+  width: 100%;
+  min-height: 28px;
+  margin-bottom: 12px;
+  overflow: hidden;
+  color: var(--card-color);
+  font-family: var(--body-alt-font-family);
+  font-weight: var(--body-font-weight-bold);
+  font-size: 34px;
+  line-height: 1.5em;
+  letter-spacing: 0.25px;
+  white-space: normal;
+  text-overflow: ellipsis;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  line-clamp: 2;
+}
+
+.poster {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 0;
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  width: 100%;
+  height: calc(70vh + 56px);
```

Is this for the header?

```suggestion
  height: calc(70vh + variables.$header-height);
```